GPU/GPUTracking/DataTypes/GPUTRDInterfaceO2Track.h

```diff
@@ -74,6 +75,11 @@ class trackInterface<o2::track::TrackParCov> : public o2::track::TrackParCov
 
   GPUdi() bool CheckNumericalQuality() const { return true; }
 
+  GPUdi() uint16_t getPileUpDistance() const { return getUserField(); }
+  GPUdi() void setPileUpDistance(uint16_t d) { return setUserField(d); }
+  GPUdi() float getPuleUpTimeShiftMUS() const { return getPileUpDistance() * o2::constants::lhc::LHCBunchSpacingMUS * 0.5; }
+  GPUdi() float getPuleUpTimeErrorMUS() const { return getPileUpDistance() * o2::constants::lhc::LHCBunchSpacingMUS / 3.4641016; }
```

Comment:
typos in the 2 lines above: getPuleUpTime... -> getPileUpTime
and why is the error divided by 3.46?

Reply:
Not sure what do you mean by typos, these are conversions of NBC range to time in \mus and its error (3.4641016 = sqrt(12)). There was a typo in the `void setPileUpDistance` (return not needed).

Reply:
The `return` I overlooked. I just meant that you put "PuleUp" instead of "PileUp", so really just typo